consensus/src/simplex/mod.rs

```diff
@@ -221,6 +221,7 @@ mod tests {
                 runtime.clone(),
                 Config {
                     registry: Arc::new(Mutex::new(Registry::default())),
+                    max_size: 1024 * 1024,
```

Comment:
Specifying `max_size` globally is also the approach taken by `p2p::authenticated`. Glad we could align this behavior.